packages/desktop/src/renderer/app/components/menus/routes-menu/routes-menu.component.html

```diff
@@ -270,6 +273,13 @@
             >
               <app-svg icon="low_priority"></app-svg>
             </div>
+            <div
+              *ngIf="route.type === 'ws'"
+              class="text-primary pe-2 d-flex align-items-center"
+              ngbTooltip="WebSocket Route"
+            >
+              <app-svg icon="websocket"></app-svg>
+            </div>
```

Comment:
I would prefer that we reuse the badge like GET, CRUD, etc with "WS", instead of an icon.
In the style.css $route-badges variables, we could use $indigo that is not yet used.